journal-of-porphyrins-and-phthalocyanines.csl

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <info>
+    <title>Journal of Porphyrins and Phthalocyanines</title>
+    <title-short>J. Porphyrins Phthalocyanines</title-short>
+    <id>http://www.zotero.org/styles/journal-of-porphyrins-and-phthalocyanines</id>
+    <link href="http://www.zotero.org/styles/journal-of-porphyrins-and-phthalocyanines" rel="self"/>
+    <link href="http://www.zotero.org/styles/histopathology" rel="template"/>
+    <link href="http://jpp.fontismedia.com/jpp/guides/Notice_to_Authors.pdf" rel="documentation"/>
+    <author>
+      <name>Jeremy Kuhne</name>
+      <email>jakuhne@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="chemistry"/>
+    <issn>1088-4246</issn>
+    <eissn>1099-1409</eissn>
+    <summary>In agreement with JPP http://jpp.fontismedia.com/jpp/guides/Notice_to_Authors.pdf</summary>
+    <published>2018-05-25T02:40:07+00:00</published>
+    <updated>2018-05-25T03:40:07+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="page-range-delimiter">-</term>
+      <term name="available at">available from</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <group suffix=".">
+      <names variable="author">
+        <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="18" et-al-use-last="true" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
What are the et al rules here? You have 19/18 here and then 4/3 below and you also use the `et-al-use-last`, which would be unusual for chemistry.

Reply:
The 18/19 is the limit on Mendeley's visual editor.  I definitely didn't want a limit and I think it got cranky when I tried turning the et-al-use-last off. Something like that. Thank you.

Reply:
So just to clarify, the style should never use et-al in the bibliography?

Reply:
The guidelines don't explicitly say so but looking at a published review it looks like et-al is never used. 

"Journals should be cited in the following way: author surnames and initials (no dot after initial), insert “and” between the last two authors (with no comma), dot, journal name (italicized and abbreviated in accordance with the ACS Style Guide..."